ncm-spma/src/main/pan/components/spma/schema.pan

```diff
@@ -42,7 +42,7 @@ type SOFTWARE_REPOSITORY = {
     "gpgcheck" : boolean = false
     "excludepkgs" ? string[]
     "includepkgs" ? string[]
-    "name" ? string  # "Repository name"
+    "name" ? string with match(SELF, '^[A-Za-z0-9_-]+$') # "Repository name"
```

Comment:
use `'^\S+$` to exclude withespace; or `^[\w-]+$` for current regexp (but no dots either?)